src/renderer/components/ConnectTroubleshooting.js

```diff
@@ -38,12 +41,9 @@ const ConnectTroubleshooting = ({ appearsAfterDelay = 25000, onRepair }: Props)
           <Trans i18nKey="connectTroubleshooting.desc" />
         </Text>
       </Box>
-      <Box horizontal>
-        <Box marginRight={100} justifyContent="center">
```

Comment:
If I recall this correctly it was something to do with the way the illustrations were shown for the blue, can you check that it shows centered with the three devices? 🤔

Reply:
I checked with @valpinkman it doesn't have anything to do with the blue animation it was just a weird flex implementation.
However the blue animation is indeed broken here but we should fix this after @gre big rework on the device connection.

Reply:
Awesome then 🚀